pygeoapi/openapi.py

```diff
@@ -275,85 +275,88 @@ def get_oas_30(cfg):
     }
 
     LOGGER.debug('setting up processes')
-    for k, v in cfg['processes'].items():
-        p = load_plugin('process', v['processor'])
+    if 'processes' in cfg:
```

Comment:
Why not the same convention as in `api.py`: `processes = cfg.get('processes', {})` and `if processes:` etc ?